pkg/setting/setting.go

```diff
@@ -609,6 +612,9 @@ func NewConfigContext(args *CommandLineArgs) error {
 	AlertingEnabled = alerting.Key("enabled").MustBool(true)
 	ExecuteAlerts = alerting.Key("execute_alerts").MustBool(true)
 
+	explore := Cfg.Section("explore")
+	ExploreEnabled = explore.Key("enabled").MustBool(true)
```

Comment:
the default value should be false. as in the config.
the default value for booleans should always be false since its the default init value for booleans.